sonar-scanner-engine/src/main/java/org/sonar/scanner/report/ComponentsPublisher.java

```diff
@@ -149,6 +150,10 @@ private static String getLanguageKey(Resource r) {
 
   @CheckForNull
   private static String getName(Resource r) {
+    if (ResourceUtils.isProject(r)) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "ResourceUtils"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)